cypress/integration/jsPromisesPracticeDOM.spec.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+Cypress.Commands.add('clickButton',
+  (mouseButton) => {
+    cy.get('body').trigger('mousedown', mouseButton)
+      .trigger('mouseup');
+  });
+
+Cypress.Commands.add('checkPromise',
```

Comment:
checkPromise - need a better name. what exactly are we checking?

Reply:
Maybe 'findPromise' or 'checkPromiseExists'